### PR TITLE
mpfs/opnsbi: refine config for for insufficient envm capacity

### PR DESCRIPTION
**Pull request by gneworld**
## Summary

to fix ci error about https://github.com/apache/nuttx/pull/14293

## Impact
mpfs/opnsbi

## Testing
ci

### CODE REVIEW

File: boards/risc-v/mpfs/icicle/configs/opensbi/defconfig [hunk 1]
```diff
@@ -53,51 +53,8 @@ CONFIG_MPFS_HART4_SBI=y
 CONFIG_MPFS_OPENSBI=y
 CONFIG_MPFS_UART0=y
 CONFIG_NSH_ARCHINIT=y
-CONFIG_NSH_DISABLE_BASENAME=y
```
Comment (GUIDINGLI) [inline boards/risc-v/mpfs/icicle/configs/opensbi/defconfig:56]: remove this will use more flash

**Comment by nuttxpr** [comment]
[**\[Experimental Bot, please feedback here\]**](https://github.com/search?q=repo%3Aapache%2Fnuttx+13552&type=issues)



This PR summary does not meet the NuttX requirements. 

**Here's what's missing:**

* **Summary:** 
    * **Insufficient detail:** "to fix ci error about https://github.com/apache/nuttx/pull/14293" is too vague.  Explain the actual error the PR addresses, not just its origin.
    * **Missing elements:** 
        * What functional part of the code (mpfs/opnsbi) is changed, and *how*? 
        * How *exactly* does this change fix the CI error?

* **Impact:** 
    * All sections are marked as "mpfs/opnsbi" which doesn't describe impact, only the area affected.  
    * Each "Impact" section needs a YES/NO answer, and a description *if* YES.  For example:
        * **Impact on build:**  NO 
        * **Impact on user:** YES (Users utilizing mpfs/opnsbi functionality may see [describe specific behavioral changes]) 

* **Testing:**
    * **Insufficient detail:** Simply stating "ci" is not enough.
    * **Missing elements:**
        * List the *specific* build hosts and targets you tested on (OS, architecture, board, configuration).
        * Provide *actual* testing logs (before and after the change) demonstrating the error and its resolution.

**Recommendation:**

Expand your PR description substantially to meet the NuttX requirements.  Provide clear, detailed information for each section to help reviewers understand and evaluate your changes.

File: boards/risc-v/mpfs/icicle/configs/opensbi/defconfig (hunk 1 above)
Comment (xiaoxiang781216) [reply boards/risc-v/mpfs/icicle/configs/opensbi/defconfig:56]: CONFIG_DEFAULT_SMALL disable all tools by default.

Comment (GUIDINGLI) [reply boards/risc-v/mpfs/icicle/configs/opensbi/defconfig:56]: ok

**Review by GUIDINGLI — APPROVED** [review]

**Pull request merged**